src/ol/render/canvas/CurveBuilder.js

```diff
@@ -76,6 +76,7 @@ class CanvasCurveBuilder extends CanvasBuilder {
     }
     this.setFillStrokeStyles();
     this.beginGeometry(curvePolygonGeometry, feature);
+    this.instructions.push(beginPathInstruction);
```

Comment:
Ik vraag mij af, werkt het ook als je deze regel weglaat? Zeker in het geval van een curve polygon met daarin gaten had ik nl. verwacht dat er per ring een `beginPath` instructie werd toegevoegd. Hij doet, als ik de code (onze eigen code :sweat_smile:) goed begrijp wel vaker een `closePath` instructie toevoegen afhankelijk van het aantal ringen/gaten.

Reply:
Oké, volgens deze StackOverflow is dat wel oké zo. Dus één `beginPath`, meerdere `closePaths`: https://stackoverflow.com/questions/13618844/polygon-with-a-hole-in-the-middle-with-html5s-canvas

Dus, deze wijziging vind ik wel logisch eigenlijk.